README.md

```diff
@@ -40,3 +40,22 @@ jobs:
             curl --fail -X PUT -F "rockspec=@-;filename=$ROCK_NAME-$TRAVIS_TAG-1.rockspec"
               https://$ROCKS_USERNAME:$ROCKS_PASSWORD@$ROCKS_SERVER
 ```
+
+## Gitlab CI integration
+
+Add `ROCKS_USERNAME`, `ROCKS_PASSWORD`, `ROCKS_SERVER` build variables.
```

Comment:
+ ROCK_NAME

Reply:
Should we get rid of ROCKS_SERVER parametrization? Initially I add it just to test demo rocks server